fjord/alerts/models.py

```diff
@@ -69,6 +79,28 @@ def __unicode__(self):
         return self.url
 
 
+class UTCDateTimeField(fields.DateTimeField):
+    """Like DateTimeField, except it always in UTC"""
```

Comment:
Actually, I'm trying out docstrings out of East Orange, NJ.

"You down with UTC? YEAH YOU KNOW ME!"